MC/bin/o2dpg_sim_workflow.py

```diff
@@ -988,13 +1010,36 @@ def getDigiTaskName(det):
      workflow['stages'].append(tpcclus)
      tpcreconeeds.append(tpcclus['name'])
 
+   # ===| TPC reco |===
+   tpcLocalCFreco=dict()
+
+   # handle distortion corrections and scaling using MC maps
+   # this assumes the lumi inside the maps is stored in FT0 (pp) scalers
+   # in case of PbPb the conversion factor ZDC ->FT0 (pp) must be set
+   tpc_corr_options_mc=''
+
+   if tpcDistortionType == 0: # disable distortion corrections
+      tpc_corr_options_mc=' --corrmap-lumi-mode 0 '
+      tpcLocalCFreco['TPCCorrMap.lumiInst'] = '-1';
```

Comment:
Disabling corrections now goes via `TPCCorrMap.lumiMean=-1`

Reply:
Thanks for spotting!